Sources/Networking/Responses/ProductsEntitlementsResponse.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ProductsEntitlementsResponse.swift
+//
+//  Created by Nacho Soto on 3/17/23.
+
+import Foundation
+
+struct ProductsEntitlementsResponse {
```

Comment:
I don't love this name, but changing it is easy, just wanted to get this blueprint structure out.